src/components/reusableComponents/BlogCard/index.tsx

```diff
@@ -1,40 +1,44 @@
-import React from "react";
-import { IBlog } from "../../../interfaces";
-import { Image } from "../Image";
-import "./blogCard.scss";
+import React from 'react'
+import { IBlog } from '../../../interfaces'
+import { Image } from '../Image'
+import './blogCard.scss'
 
 interface IBlogCardProps {
-  blogData: IBlog;
+  blogData: IBlog
   onClickCard: (path: string) => void
 }
 
 export const BlogCard = (props: IBlogCardProps) => {
   return (
-    <div className="card-container">
-      <div className="card-image">
+    <div className='card-container'>
+      <div className='card-image'>
         <Image
           desktop_src={props.blogData?.banner_small?.url}
           mobile_src={props.blogData?.mobile_card?.url}
         />
-        <span className="env">Devops</span>
+        <span className='env'>{props?.blogData?.env}</span>
```

Comment:
@jyotik1398 env does not hold the string we want to display at the chip. Let's discuss this.